.github/workflows/analyze.yml

```diff
@@ -44,11 +44,20 @@ jobs:
 
       - run: dart fix --apply
 
+      - name: swiftlint --fix
+        uses: norio-nomura/action-swiftlint@9f4dcd7fd46b4e75d7935cf2f4df406d5cae3684 # pin@v3.2.1
+        if: ${{ inputs.package == 'flutter' }}
+        with:
+          args: --strict --fix
+
         # actions/checkout fetches only a single commit in a detached HEAD state. Therefore
         # we need to pass the current branch, otherwise we can't commit the changes.
         # GITHUB_HEAD_REF is the name of the head branch. GitHub Actions only sets this for PRs.
-      - run: ../scripts/commit-formatted-code.sh $GITHUB_HEAD_REF
-        if: env.GITHUB_HEAD_REF != null
+      - name: commit formatted code
```

Comment:
If this is not working, lets revert the format step and fix the lining manually, raise a new issue for this.